### ISSUE TITLE
Hyperlink References

### ISSUE THREAD

**Issue by langdonholmes** [learlab/itell#11]
These pages include references as footnotes. The in-text citations are numbered references, e.g., [1].

The numerical in-text reference should be hyperlinked to the footnote.

@wesleygmorris Also pointed out that the summary task appears below a potentially long (50+) list of references. I would suggest moving the references to a separate page.

I am promoting this to an issue.

**Comment by jchoi92k** [comment]
Agreed on all points. Not a fan of the references taking up half the page. But not sure if we can make the changes on time (separate page for references, hyperlinks to footnotes, etc.). 

Seems like what Scott had in mind was to have the references inside green/gray callout boxes. I suggest for now we do the callout boxes, and make more changes if we can get to them on time.
